packages/components/pie-webc-core/src/test/mixins/formControlMixin/formControlMixin.browser.spec.ts

```diff
@@ -0,0 +1,99 @@
+import { test, expect } from '@sand4rt/experimental-ct-web';
+import { MockComponent } from './MockComponent.ts';
+
+test.describe('FormControlMixin', () => {
+    // IMPORTANT: Mounting and Unmounting the component before each test ensures that any tests that do not explicitly
+    // mount the component will still have it available in Playwright's cache (loaded and registered in the test browser)
+    test.beforeEach(async ({ mount }) => {
+        await (await mount(MockComponent)).unmount();
+    });
+
+    test.describe('form property', () => {
+        test('should not have an associated form when not inside of a form', async ({ mount }) => {
+            // Arrange
+            const component = await mount(
+                MockComponent,
+                {
+                    props: {},
+                },
+            );
+
+            // Assert
+            expect(component).not.toBeNull();
```

Comment:
If it is not null it might be worth adding the expected output here.

Reply:
This was a mistake - the test has been revamped :)